nomos-services/blend/src/backends/libp2p.rs

```diff
@@ -96,43 +98,72 @@ impl BlendBackend for Libp2pBlendBackend {
     }
 }
 
-struct BlendSwarm {
-    swarm: Swarm<nomos_blend_network::Behaviour<SphinxMessage, TokioIntervalStreamProvider>>,
+struct BlendSwarm<R> {
+    swarm: Swarm<BlendBehaviour>,
     swarm_messages_receiver: mpsc::Receiver<BlendSwarmMessage>,
     incoming_message_sender: broadcast::Sender<Vec<u8>>,
+    // TODO: Instead of holding the membership, we just want a way to get the list of addresses.
+    membership: Membership<PeerId, SphinxMessage>,
+    rng: R,
+    peering_degree: usize,
```

Comment:
I'll update this to `u16` by rebasing this branch to the https://github.com/logos-co/nomos-node/pull/987, after all changes are reviewed completely.